node/src/services/mod.rs

```diff
@@ -61,13 +58,8 @@ impl<S: StorageHubHandlerConfig> StorageHubHandler<S> {
     }
 
     pub fn start_bsp_tasks(&self) {
-        ResolveRemoteUploadRequest::new(self.clone())
-            .subscribe_to(&self.task_spawner, &self.file_transfer)
-            .start();
-        NewStorageRequestHandler::new(self.clone())
-            .subscribe_to(&self.task_spawner, &self.blockchain)
-            .start();
-        AcceptedBspVolunteerHandler::new(self.clone())
+        log::info!("Starting BSP tasks");
+        BspVolunteerMockTask::new(self.clone())
```

Comment:
Minor thing but I would leave a TODO here just in case, it will definitely not happen but if we are super SUPER distracted we might miss this mock and leave it there in prod 😬